src/profile-logic/import/instruments/index.js

```diff
@@ -0,0 +1,899 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+// @flow
+
+//types
+import type {
+  Profile,
+  FuncTable,
+  FrameTable,
+  StackTable,
+} from '../../../types/profile';
+
+type Sample = {|
+  +timestamp: number,
+  +threadID: number,
+  +backtraceID: number,
+  backtraceStack?: Array<number>,
+|};
+
+type TraceDirectoryTree = {|
+  +name: string,
+  +files: Map<string, File>,
+  +subdirectories: Map<string, TraceDirectoryTree>,
+|};
+
+type FrameInfo = {|
+  +key: string | number,
+  +name: string,
+  file?: string,
+  +line?: number,
+  +col?: number,
+|};
+
+type FormTemplateRunData = {|
+  +number: number,
+  +addressToFrameMap: Map<number, FrameInfo>,
+|};
+
+import type { UniqueStringArray } from '../../../../src/utils/unique-string-array';
+
+//utils
+import {
+  getEmptyProfile,
+  getEmptyThread,
+} from '../../../profile-logic/data-structures';
+import BinaryPlistParser, { UID } from './BinaryPlistParser';
+import BinReader from './BinReader';
+import MaybeCompressedReader from './MaybeCompressedReader';
```

Comment:
Here, we have some imports, then some type definitions and then some more imports. Can we seperate imports and type definitions? I'm using that order usually, and I think that is the most used order for the whole project:
```
imports
type imports
type definitions
```